cfgov/unprocessed/css/enhancements/forms.less

```diff
@@ -25,7 +25,7 @@ select[multiple] {
     - cf-forms
 */
 
-@input-warning:     @gold;
+/*@input-warning:     @gold;
```

Comment:
Shoot I can't recall why I commented this out! But I can't find a use for it. Removed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/4c234e9094b3ba140bc8e5df91c154f457072e34